### PR TITLE
Corrections to the English translation, additions and corrections to the Russian translation, typos corrections

### PR DESCRIPTION
**Pull request by maximus105**
Summary
=======
1. Added translation for untranslated words in the menu into Russian.
2. Added translation for the word "Frequency" for all language files.
3. Corrections of errors in translation.
4. Corrected the word "Speed" for processors to the word "Frequency" for all language files. At the same time, the word "Speed" for HDD and CD-ROM remains as before.
5. Added the word "3Dfx" for Voodoo Graphics.
6. Word corrections for IBM 8514/A Graphics and XGA Graphics.
7. Word corrections for OpenGL 3.0 Renderer options.
8. Word corrections for Joystick Configuration.
9. Corrected the word "3dfx" to "3Dfx".
10. Added the word "3Dfx" for Voodoo video cards.
11. Corrected the word "max. speed" to "Max. Speed" for HDD.

Checklist
=========
* [ ] Closes #xxx
* [ ] I have discussed this with core contributors already
* [ ] This pull request requires changes to the ROM set
  * [ ] I have opened a roms pull request - https://github.com/86Box/roms/pull/changeme/

References
==========
_Provide links to datasheets or other documentation that helped you implement this pull request._

### CODE REVIEW

File: src/qt/languages/ru-RU.po [hunk 1]
```diff
@@ -25,11 +25,17 @@ msgstr "&Выход..."
...
+msgid "Hide &toolbar"
+msgstr "С&крыть панель инструментов"
+
 msgid "&Hide status bar"
 msgstr "&Скрыть строку состояния"
 
-msgid "Hide &toolbar"
-msgstr "С&крыть панель инструментов"
+msgid "Show status icons in fullscreen"
+msgstr "Показывать значки статуса в полноэкранном режиме"
```
Comment (binarymaster) [inline src/qt/languages/ru-RU.po:35]:
```suggestion
msgstr "Показывать значки состояния в полноэкранном режиме"
```
(consistency with `msgid "&Hide status bar"`)

File: src/qt/languages/ru-RU.po [hunk 2]
```diff
@@ -50,13 +56,37 @@ msgid "SDL (&OpenGL)"
...
 
 msgid "Open&GL (3.0 Core)"
-msgstr "Open&GL (3.0)"
+msgstr "Open&GL 3.0"
 
 msgid "&VNC"
 msgstr "&VNC"
 
+msgid "Renderer options..."
+msgstr "Параметры рендеринга OpenGL 3.0..."
```
Comment (binarymaster) [inline src/qt/languages/ru-RU.po:65]:
```suggestion
msgstr "Параметры рендеринга..."
```
Probably a copypaste, check original msgid.

File: src/qt/languages/ru-RU.po [hunk 3]
```diff
@@ -127,6 +157,9 @@ msgstr "&Целочисленное масштабирование"
 msgid "4:&3 Integer scale"
 msgstr "4:&3 Целочисленное масштабирование"
 
+msgid "Apply fullscreen stretch mode when maximized"
+msgstr "Применить полноэкранный режим растяжения при максимальном увеличении"
```
Comment (binarymaster) [inline src/qt/languages/ru-RU.po:161]:
```suggestion
msgstr "Применить полноэкранный режим растяжения при разворачивании окна"
```
"maximized" is a term for the window state.

File: src/qt/languages/ru-RU.po [hunk 4]
```diff
@@ -232,6 +268,9 @@ msgstr "&Перемотка в конец"
 msgid "E&ject"
 msgstr "И&звлечь"
 
+msgid "Eject %s"
+msgstr "И&звлечь"
```
Comment (binarymaster) [inline src/qt/languages/ru-RU.po:272]:
```suggestion
msgstr "Извлечь %s"
```
Check original msgid.

**Review by binarymaster — COMMENTED** [review]
Reviewed ru-RU.po

File: src/qt/languages/ru-RU.po (hunk 4 above)
Comment (maximus105) [reply src/qt/languages/ru-RU.po:272]: This line was added for Floppy. To apply the translation for the word "Eject" (Under the line "Export to 86F").

**Comment by OBattler** [comment]
Please correct the relevant strings in the other language files as well, or the other translations are going to be broken.

**Comment by maximus105** [comment]
> Please correct the relevant strings in the other language files as well, or the other translations are going to be broken.

Corrected the word "Speed" for processors to the word "Frequency" for all language files, added a translation to the word "Frequency" for all language files

**Pull request closed without merging**